pkg/util/validation/limits.go

```diff
@@ -168,6 +169,8 @@ type Limits struct {
 	ForwardingEndpoint      string          `yaml:"forwarding_endpoint" json:"forwarding_endpoint" doc:"nocli|description=Remote-write endpoint where metrics specified in forwarding_rules are forwarded to. If set, takes precedence over endpoints specified in forwarding rules."`
 	ForwardingDropOlderThan model.Duration  `yaml:"forwarding_drop_older_than" json:"forwarding_drop_older_than" doc:"nocli|description=If set, forwarding drops samples that are older than this duration. If unset or 0, no samples get dropped."`
 	ForwardingRules         ForwardingRules `yaml:"forwarding_rules" json:"forwarding_rules" doc:"nocli|description=Rules based on which the Distributor decides whether a metric should be forwarded to an alternative remote_write API endpoint."`
+
+	EphemeralSeriesMatchers ephemeral.LabelMatchers `yaml:"ephemeral_series_matchers" json:"ephemeral_series_matchers" doc:"nocli|description=Matchers to check if a series should be marked ephemeral, matching series get marked to be stored in the ephemeral store." category:"experimental"`
```

Comment:
Why the "description" here? If we set the description here, the CLI flag description won't be used, but CLI flag description is more detailed.

Reply:
> Why the "description" here? 

I just followed the example of the line above.

I removed the description here: 3635f97f3

Reply:
I'm trying to figure out how to make it use the description from the CLI flag.
Do you know if there's something else that I need to do apart from just removing this `doc` tag?